commands/assign-dish.js

```diff
@@ -0,0 +1,35 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { getMenu, saveMenu } = require('./../menu-memory/menu.js');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('assign-dish')
+		.setDescription('assign a dish to a user')
+		.addUserOption(option => option.setName('user').setDescription('Who is bringing the dish>'))
+		.addStringOption(option => option.setName('dish').setDescription('Enter a dish')),
+	async execute(interaction) {
+		let user = interaction.options.getUser('user');
+		const dish = interaction.options.getString('dish');
+		let menu = await getMenu();
+		
+		if (dish == undefined) {
+			return interaction.reply(`No dish was provided to sign up for`);
+		}
+		
+		if (menu.Dishes[dish] != undefined && menu.Dishes[dish] != "") {
+			return interaction.reply(`${dish} is already being brought by <@${menu.Dishes[dish]}>`)
+		}
+		
+		// Only the guy in charge can assign other people dishes
```

Comment:
Let's keep our comments gender neutral

Reply:
Good point.
```suggestion
		// Only the member in charge can assign other people dishes
```

Reply:
Let me resolve my own comments when I'm satisfied with the result 😛

Reply:
Write a Comtrivuting.md for the repo/org so there are rules ;P